examples/streampay/zillabase/migrations/000001__create_commands_and_replies.sql

```diff
@@ -1,18 +1,27 @@
 -- create_commands_and_replies
 
 -- A stream to track all of the user commands for the application
-CREATE STREAM streampay_commands(
+CREATE OR REPLACE FUNCTION compute_status_and_correlation_id(record RECORD)
+RETURNS RECORD AS $$
+BEGIN
+    record.status := CASE
+        WHEN record.type IN ('SendPayment', 'RequestPayment', 'RejectRequest') THEN '200'
+        ELSE '400'
+    END;
+    record.correlation_id := encode(record.zilla_correlation_id, 'escape');
+    RETURN record;
```

Comment:
Does `record` mean two different things here, i.e. the `command` and also the `reply` or am I misreading the code?

Reply:
`record` refers to the same variable both when you receive it as a parameter and when you modify and return it.